products/serializers.py

```diff
@@ -0,0 +1,58 @@
+from rest_framework import serializers
+from products.models import Category, Brand, Product, Attribute, ProductAttribute
+
+
+class AttributeSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Attribute
+        fields = ('id', 'name')
+
+
+class ProductAttributeSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = ProductAttribute
+        fields = (
+            'id',
+            'product',
+            'attribute',
+            'value'
+        )
+
+
+class ProductSerializer(serializers.ModelSerializer):
+    product_attributes = ProductAttributeSerializer(many=True)
```

Comment:
Is "product_attributes" field for reading or for writing? I suppose we need "read_only=True" here